understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
-                  page. Items which are after the help item on other pages should be after the help
-                  item on this page.
+               <p>この達成基準をテストする場合、残りのコンテンツと関連するヘルプ項目になる。ページをテストする場合、ウェブページ一式の全体に存在して、ヘルプ項目の前にある他のコンテンツは、このページのヘルプ項目の前に存在すべきである。他のページでヘルプ項目の後にある項目は、このページのヘルプ項目の後に存在すべきである。
                </p>
-               <p>If the help item is visually in a different location, but in the same serial order,
-                  that is not helpful from a user's point of view, but it would not fail this criterion.
+               <p>ヘルプ項目が視覚的には異なる場所にあるものの、同じシリアル順序で配置されている場合、利用者の観点からは役に立たないが、この基準に失敗することにはならない。
                </p>
-               <p>When having problems completing a task on a Web site (or part of a Web site, what
-                  we call a <a href="#dfn-set-of-web-pages">set of Web pages</a>), people with some types of disabilities may not be able to work through the issue
-                  without further help.  Issues could include difficulty:
-                  completing a form, or finding a document or page which provides information required
-                  to complete a task.
+               <p>ウェブサイト (又はウェブサイトの一部、いわゆる<a href="#dfn-set-of-web-pages">ウェブページ一式</a>) でタスクの完了に問題がある場合、障害のある人の中には、さらなるヘルプなしに問題を解決できない場合がある。問題には、フォームの完了、又は作業を完了するために必要な情報を提供する文書もしくはページを見つけることの難しさが含まれる可能性がある。
                </p>
-               <p>Without help, some users may abandon the task. They may also fail to correctly complete
-                  a task, or they may require assistance from people who do not necessarily keep private
-                  information secure.
+               <p>ヘルプがなければ、一部の利用者は作業をあきらめる可能性がある。また、作業を正しく完了するのに失敗する場合、又は個人情報を安全に管理しているとは限らない人からの支援を必要とする場合もある。
                </p>
                <section id="limitations">
                   
-                  <h3>Limitations and Exceptions</h3>
+                  <h3>制限及び例外</h3>
                   
                   
-                  <p>It is not the intent of this Success Criterion to require authors to provide help
-                     or access to help. The Criterion only requires that <em>when</em> one of the listed forms of help is available across multiple pages that it be in
-                     a consistent location. It does not require authors to provide help information on
-                     PDFs or other static documents that may be available for viewing/download from the
-                     Web pages. PDFs and other static documents are not considered part of the "<a href="#dfn-set-of-web-pages">set of web pages</a>" from which they are downloaded.
+                  <p>この達成基準の意図は、コンテンツ制作者にヘルプの提供又はヘルプへのアクセスを要求することではない。この基準は、リストされた形式のヘルプの一つが複数のページにわたって利用できる<em>場合に</em>、そのヘルプが一貫した場所にあることだけを要求する。ウェブページから表示／ダウンロードできる PDF 又はその他の静的ドキュメントに関するヘルプ情報をコンテンツ制作者が提供する必要はない。PDF 及びその他の静的ドキュメントは、ダウンロード元の「<a href="#dfn-set-of-web-pages">ウェブページ一式</a>」の一部とはみなされない。
```

Comment:
「help information **on** PDFs or other static documents」なので、「〜に関する」よりは「〜における」という感じのように思いました。以下はいかがでしょうか？

```suggestion
                  <p>この達成基準の意図は、コンテンツ制作者にヘルプの提供又はヘルプへのアクセスを要求することではない。この基準は、リストされた形式のヘルプの一つが複数のページにわたって利用できる<em>場合に</em>、そのヘルプが一貫した場所にあることだけを要求する。ウェブページから表示／ダウンロードできる PDF 又はその他の静的ドキュメントにおいてはヘルプ情報をコンテンツ制作者が提供する必要はない。PDF 及びその他の静的ドキュメントは、ダウンロード元の「<a href="#dfn-set-of-web-pages">ウェブページ一式</a>」の一部とはみなされないからである。
```

Reply:
ここのonはaboutと同じように取ってしまって問題ないと思っています。
文末を変えるとしたらこうでしょうか？

```suggestion
                  <p>この達成基準の意図は、コンテンツ制作者にヘルプの提供又はヘルプへのアクセスを要求することではない。この基準は、リストされた形式のヘルプの一つが複数のページにわたって利用できる<em>場合に</em>、そのヘルプが一貫した場所にあることだけを要求する。ウェブページから表示／ダウンロードできる PDF 又はその他の静的ドキュメントに関するヘルプ情報をコンテンツ制作者が提供する必要はない。PDF 及びその他の静的ドキュメントは、ダウンロード元の「<a href="#dfn-set-of-web-pages">ウェブページ一式</a>」の一部とはみなされることはない。
```

Reply:
改めて読み返してみました。なるほど、そうですね。問題ないと思います！